web-ui/src/app/components/analysis/analysis.component.ts

```diff
@@ -96,6 +101,12 @@ export class AnalysisComponent implements OnInit, OnDestroy {
         this.showMoreSubject.next(this.showMoreSubject.value + 1);
     }
 
+    getWarningMessage() {
```

Comment:
It's actually not relevant for this component: the `AnalysisStep` is the last step and nothing can cause the state be marked invalid. It must be implemented though because it is marked `abstract` in the parent `StepComponent`.